services/grafana-logging/8.3.6/defaults/cm.yaml

```diff
@@ -28,10 +28,11 @@ data:
       pathType: ImplementationSpecific
 
     sidecar:
-      sidecar:
-        image:
-          repository: ghcr.io/mesosphere/dkp-container-images/docker.io/kiwigrid/k8s-sidecar
-          tag: 1.25.3-d2iq.0
+      image:
+        registry: ghcr.io
+        repository: mesosphere/dkp-container-images/quay.io/kiwigrid/k8s-sidecar
+        tag: 1.26.1-d2iq.1
```

Comment:
Is this correct? Why we didn't need to split registry/repository in existing version?